lib/pages/chat/chat.dart

```diff
@@ -900,7 +853,7 @@ class ChatController extends State<Chat>
     );
     if (highlight) {
       await scrollController.highlight(
-        index,
+        index + 1,
```

Comment:
Do the logic out of the scroll function.

Reply:
done